app/auth/login/page.tsx

```diff
@@ -0,0 +1,79 @@
+
+import { getFormData } from '@/lib/login';
+
+export default function LoginPage() {
```

Comment:
This page looks good. But this is at the top of the page. Lets try to center it in the middle of the page